release-team/role-handbooks/enhancements/README.md

```diff
@@ -83,26 +83,21 @@ It is important that this process be followed and documentation remain up-to-dat
-- Send an email to the [Kubernetes-Dev](https://groups.google.com/forum/#!forum/kubernetes-dev) mailing list with a call for enhancements. View prior examples for references ([1](https://groups.google.com/forum/#!msg/kubernetes-dev/vGSBzUUWp9g/bFa1re3ZAgAJ) [2](https://groups.google.com/forum/#!msg/kubernetes-dev/T-kIHtgS5J4/ZjKTsCf_AwAJ;context-place=msg/kubernetes-dev/vGSBzUUWp9g/bFa1re3ZAgAJ)).
 
 #### Pre-Enhancements Freeze (Week 1)
 
 - Gather Shadows to have them read this handbook, watch the [k/enhancements][k/enhancements] repo to get notified on all issues, and give expectations on what the process looks like and their particular role. If possible, try to schedule a call with the shadows to get them accustomed to the team. This helps as a great team building exercise.
-- Go to every open issue that is marked with `tracked/no` and ping the owner to see if the enhancement should be included in the current milestone. If there is no graduation or major change, leave as-is. If the enhancement is graduating or undergoing a major change, remove the `tracked/no` label and add `tracked/yes`, add the issue to the current milestone, and set it's status to `Tracked` within the tracking sheet if it meets the requirements. If it is missing some  of the requirements (e.g. KEP in a provisional state), let the issue owner know and add it as `At Risk`. Another great task for shadow help or some automation.
+- Send an email to the [Kubernetes-Dev](https://groups.google.com/forum/#!forum/kubernetes-dev) mailing list and a message to #chairs-and-techleads slack channel with a call for enhancements and how to opt-in to the release.
 - Verify issues have k/k PRs associated so they can be referenced and easily tracked. This is going to be critical come Enhancement Freeze and Code Freeze to see the status of the code.
-- Work with the Release Lead to attend every SIG meeting over the span of 2-3 weeks. Take 2 minutes to introduce yourself, talk about release information, and relay information about issues that haven't been removed from the previous milestone or call out issues that may need some attention.
-- New issues may be added during the release cycle. Take the opportunity to apply the correct labels, milestone information, and add them to the `data` tab within the sheet. If it's not being tracked for the current release, adding the information will help a later release team.
+- Work with the Release Lead to attend every SIG meeting over the span of 2-3 weeks. Take 2 minutes to introduce yourself, talk about release information, and relay information about opting into the release.
```

Comment:
This seems like a heavy and unnecessary burden, clearly communicating the process to the leads should allow them to disseminate the info to their sigs.  I think an email to sig-leads is sufficient on this along with a warm intro in their slack channel/mailing list.

Reply:
I'm a little confused because this was already in the handbook. The only change I made was adding the "relay information about opting into the release" part. Are you saying that this has not been done in the past release and we should remove it from the handbook?

tbh, I was a bit surprised reading this step and agree that it is a big burden to try to attend all sig meetings.

Reply:
yes exactly. I would remove and change to:

"Work with the Release Lead to introduce yourself, talk about release information, and relay information about opting into the release with SIG Leads."

Reply:
done!